components/widgets/forest-change/glad-alerts-daily/selectors.js

```diff
@@ -0,0 +1,204 @@
+
+    const end = endIndex && (currentData.length - 1 >= endIndex) ? endIndex : currentData.length - 1;
+
+    return {
+      startIndex: start,
+      endIndex: end
+    };
+  }
+);
+
+export const parseBrushedData = createSelector(
+  [getData, getStartEndIndexes],
+  (data, indexes) => {
+    if (!data) return null;
+    const { startIndex, endIndex } = indexes;
+
+    const start = startIndex || 0;
+    const end = endIndex || data.length - 1;
+    return data.slice(start, end + 1);
+  }
+);
+
+export const parseConfig = createSelector(
+  [getColors, getStartEndIndexes],
+  (colors, indexes) => {
+    const { startIndex, endIndex } = indexes;
+
+    const tooltip = [
+      {
+        label: 'GLAD alerts'
+      },
+      {
+        key: 'count',
+        labelKey: 'date',
+        labelFormat: value => moment(value).format('MMM DD YYYY'),
+        unit: ' GLAD alerts',
+        color: colors.main,
+        unitFormat: value =>
+          (Number.isInteger(value) ? format(',')(value) : value)
+      }
+    ];
+
+    return {
+      ...getChartConfig(colors),
+      tooltip,
+      xAxis: {
+        tickFormatter: t => moment(t).format('MMM DD, YY')
+      },
+      brush: {
+        width: '100%',
+        height: 60,
+        margin: {
+          top: 0,
+          right: 10,
+          left: 48,
+          bottom: 12
+        },
+        minimumGap: 30,
+        maximumGap: 0,
+        dataKey: 'date',
+        startIndex: startIndex || 0,
+        endIndex,
+        config: {
+          margin: {
+            top: 5,
+            right: 0,
+            left: 42,
+            bottom: 20
+          },
+          yKeys: {
+            lines: {
+              count: {
+                stroke: colors.main,
+                isAnimationActive: false
+              },
+              compareCount: {
+                stroke: '#49b5e3',
+                isAnimationActive: false
+              }
+            }
+          },
+          xAxis: {
+            hide: true,
+            scale: 'point'
+          },
+          yAxis: {
+            hide: true
+          },
+          cartesianGrid: {
+            horizontal: false,
+            vertical: false
+          },
+          height: 60
+        }
+      }
+    };
+  }
+);
+
+export const parseSentence = createSelector(
+  [
+    parseBrushedData,
+    getColors,
+    getSentences,
+    getLocationObject,
+    getOptionsSelected,
+    getIndicator
+  ],
+  (data, colors, sentences, location, options, indicator) => {
+    if (!data || !data.length) return null;
+    const { initial, withInd, highConfidence } = sentences;
+    const { confidence, dataset } = options;
+    const indicatorLabel =
+      indicator && indicator.label ? indicator.label : null;
+
+    const startDate = data[0].date;
+    const endDate = data[data.length - 1].date;
+    const total = sumBy(data, 'alert__count');
+
+    let sentence = indicator ? withInd : initial;
```

Comment:
HighConfidence sentence doesn't exist if we are not going to include a confirmed alerts setting. This can be removed unless we want to implement it.